src/scripts/pokemons/RoamingPokemonList.ts

```diff
@@ -38,80 +46,60 @@ class RoamingPokemonList {
         // Seed the random runmber generator
         SeededRand.seedWithDateHour(date, this.period);
 
-        this.increasedChanceRoute.forEach((route, region) => {
-            const routes = Routes.getRoutesByRegion(region);
-            // Select a route
-            const selectedRoute = SeededRand.fromArray(routes);
-            route(selectedRoute);
+        RoamingPokemonList.increasedChanceRoute.forEach((subRegionGroups, region) => {
+            subRegionGroups.forEach((route, subRegion) => {
+                const routes = Routes.getRoutesBySubRegionRoamerGroup(region, subRegion);
+                // Select a route
+                const selectedRoute = SeededRand.fromArray(routes);
+                route(selectedRoute);
+            });
         });
     }
 }
 
 // Kanto
-RoamingPokemonList.add(GameConstants.Region.kanto, new RoamingPokemon('Mew'));
+RoamingPokemonList.add(GameConstants.Region.kanto, 0, new RoamingPokemon('Mew'));
```

Comment:
Would be better to use our `KantoSubregions` variable in this file instead of magic numbers.

Reply:
Actually it's probably worth moving all the individual subregion contants to `GameConstants` as an array and handle it simillarly to `RegionDungeons`.

That way we could also remove the `TODO` above about not knowing the max subregions count.